OWNERS.md

```diff
@@ -44,8 +44,8 @@ Being part of Technical Steering Committee (TSC) [@StackStorm/maintainers](https
 
 # Contributors
 Contributors are using and occasionally contributing back to the project, might be active in conversations or express their opinion on the project’s direction.
-They're not part of the TSC voting process, but appreciated for their contribution, involvement and may become Maintainers in future.
-<!-- [@StackStorm/contributors](https://github.com/orgs/StackStorm/teams/contributors) are invited to StackStorm Github organization and we appreciate their contribution and involvement. -->
+They're not part of the TSC voting process, but appreciated for their contribution, involvement and may become Maintainers in the future depending on their effort and involvement. See [How to become a Maintainer?](https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#how-to-become-a-maintainer)
+[@StackStorm/contributors](https://github.com/orgs/StackStorm/teams/contributors) are invited to StackStorm Github organization and have permissions to help triaging the Issues and reviewing PRs.
```

Comment:
Minor comment on wording, should it be "have permissions to help triage issues and review PRs". To me, "to help triaging the Issues" doesn't read quite correctly.

Reply:
Updated in https://github.com/StackStorm/st2/pull/5019/commits/e828eb870cd5bf264b6148a3c3af229de207e684
Thanks for the fixed version! :+1: